### ISSUE TITLE
How to handle concurrent operations by multiple users

### ISSUE THREAD

**Issue by dadiorchen** [Greenstand/treetracker-admin-client#976]
For the matching tool, what will happen if multiple users open it at the same time, and do matching, the same question applies to verifying captures, right?

**Comment by dadiorchen** [comment]
@nmcharlton @sebastiangaertner @Kpoke

**Comment by nmcharlton** [comment]
@dadiorchen You're right. We don't have a way of deconflicting simultaneous verification or capture matching by two users.
@Kpoke What happens if we try to create a `capture` record for a `raw_capture` that already exists?

As for capture matching, I expect the second user will overwrite whatever the first user has done.

The `det` model proposed by Cam for species identification may be a way to solve this issue in the long term.
